frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -52,9 +57,23 @@ export default function ModelAccess({ model }: ModelAccessProps) {
     }
   }
 
-  // TODO - add a request to update the model's collaborators field
   async function updateAccessList() {
-    await patchModel(model.id, { collaborators: accessList })
+    const res = await patchModel(model.id, { collaborators: accessList })
```

Comment:
Always handle errors first and keep the indentation size of the 'success' branch linear.  For instance, in this case you could achieve that with:

```javascript
const res = await patchModel(model.id, { collaborators: accessList })

if (!res.ok) {
  return sendNotification({
    variant: 'success',
    msg: error,
    anchorOrigin: { horizontal: 'center', vertical: 'bottom' },
  })
}

sendNotification({
  variant: 'success',
  msg: 'Model access list updated',
  anchorOrigin: { horizontal: 'center', vertical: 'bottom' },
})
mutateModel()
```

This matters less for short functions like this.